examples/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+find_package(vsgXchange CONFIG REQUIRED)
```

Comment:
I don't think REQUIRED is required :-)

The nested CMakeLists check to see if vsgXchange is available and then uses it.  The .cpp's also do a conditional compile time check.